pkg/testhelpers/business_case.go

```diff
@@ -0,0 +1,44 @@
+package testhelpers
+
+import (
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// EstimatedLifecycleCostOptions allows cost options to be customized
+type EstimatedLifecycleCostOptions struct {
+	BusinessCaseID *uuid.UUID
+	Solution       *models.LifecycleCostSolution
+	Phase          *models.LifecycleCostPhase
+	Year           *models.LifecycleCostYear
+	Cost           *int
+}
+
+// NewEstimatedLifecycleCost helps generate a new lifecycle cost for a given business case
+func NewEstimatedLifecycleCost(opts EstimatedLifecycleCostOptions) models.EstimatedLifecycleCost {
+	elc := models.EstimatedLifecycleCost{
+		ID:             uuid.New(),
+		BusinessCaseID: uuid.New(),
+		Solution:       models.LifecycleCostSolutionASIS,
+		Phase:          models.LifecycleCostPhaseDEVELOPMENT,
+		Year:           models.LifecycleCostYear1,
+		Cost:           100,
+	}
+	if opts.Solution != nil {
```

Comment:
Milmove had a method called MergeModels to do this. Is that worth implementing here?

```
// mergeModels merges src into dst, if non-zero values are present
// dst should be a pointer the struct you are merging into
func mergeModels(dst, src interface{}) {
	noErr(
		mergo.Merge(dst, src, mergo.WithOverride, mergo.WithTransformers(customTransformer{})),
	)
}
```
in mymove pkg/testdatagen/shared.go

Reply:
once we are doing this more, maybe? but I'd like to know what's going on in the custom Transformer before outright saying yes.